servus/servus.cpp

```diff
@@ -32,6 +37,13 @@
 #  include <unistd.h>
 #endif
 
+#ifdef COMMON_USE_CXX11
```

Comment:
my bad. Logic is that we'll eventually have CXX14, so should always be if(03) xx else 11/14 code